### PR TITLE
NEWS: Update v1.11.2-rc1 release

### PR DESCRIPTION
**Pull request by yosefe**
## Why
Update NEWS file with the bugfixes of v1.11.2 release

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -7,8 +7,13 @@
...
 #
 
-## 1.11.2 (TBD)
-
+## 1.11.2-rc1 (September 23, 2021)
+### Bugfixes
+* Fixes in Java release pipeline
+* Fixes in handling large number of devices
+* Fixes in UD out-of-order processing
+* Fixes in switching trasports during cleint/server connection setup
```
Comment (brminich) [inline NEWS:15]: `transports`  `client`

File: NEWS [hunk 2]
```diff
@@ -7,8 +7,13 @@
...
+* Fixes in trasport-level error reporting
```
Comment (brminich) [inline NEWS:16]: `transport`

**Comment by yosefe** [comment]
@brminich fixed

**Comment by yosefe** [comment]
@shamisp can you pls take a look?

**Comment by shamisp** [comment]
👍

**Pull request merged**